BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -519,3 +512,57 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
     }
 }
 
+private extension GalleryViewController {
```

Comment:
This seems like a good candidate to be its own view, rather than an extension to `GalleryViewController`

Reply:
What did you have in mind? It is just a label with a bit of spacing, configured for being displayed in the Gallery. It is here inside this `private extension` so we can keep the details out of the higher level further up where it is used.
To better follow the file's pattern, it could be configured in the viewDidLoad.
I think if it were in SwiftUI, it would be its own named view.

Reply:
I'm pretty -1 on extensions overall - I see them as resulting in code being in multiple places, without adding anything in the way of real structure. Having said that, I'm not going to totally veto them, but as the very least each one needs a comment describing the functionality it contains.

Reply:
[Do] Add a comment summarizing the functionality encased in this extension.